internal/rbd/nodeserver.go

```diff
@@ -512,6 +512,11 @@ func resizeNodeStagePath(ctx context.Context,
 		if err != nil {
 			return status.Error(codes.Internal, err.Error())
 		}
+
+		// if this is a AccessType=Block volume, do not attempt filesystem resize
```

Comment:
Just add more details that if the filesystem is created by te user it user's responsibility to resize the filesystem.

Reply:
That should be clear if an application uses BlockMode volumes?

Reply:
i was looking for why we should not attempt for filesystem resize in this case.

Reply:
Ok, does this extra comment help?